pkg/app/api/service/webservice/service.proto

```diff
@@ -401,3 +404,13 @@ message ListAPIKeysRequest {
 message ListAPIKeysResponse {
     repeated model.APIKey keys = 1;
 }
+
+message GetApplicationDeploymentFrequencyRequest {
+    string application_id = 1 [(validate.rules).string.min_len = 1];
+    repeated int64 dates = 2;
```

Comment:
@sanposhiho If we want to fetch for a week data, how we use this parameter?
`[startDate, endDate]` or `[date1, date2, date3, ..., date7]` ?

Reply:
`[date1, date2, date3, ..., date7]`, so that we may want to add filter for dates(not including holidays in the graph, etc.) later.

Reply:
>(not including holidays in the graph, etc.) 

Agree with you 👍